spring-batch-core/src/main/resources/org/springframework/batch/core/migration/migration-hana.sql

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I'm not sure we need this "migration" script, as there was no support for SAP HANA previously. Migration scripts in the `org/springframework/batch/core/migration` package are typically for the modifications to the existing DDL scripts of an already supported database and which are required to migrate to a newer version of Spring Batch.

Reply:
I have removed the migration script.